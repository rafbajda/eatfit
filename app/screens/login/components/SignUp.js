/* eslint-disable react/no-unescaped-entities */
import React from 'react';
import { SignUpContainer, InformationText, SignUpText } from '../styles/signUpStyles';
import screens from '../../../navigation/screens';

const SignUp = props => {
    const { nav } = { ...props };
    return (
        <SignUpContainer>
            <InformationText>
                Don't have an account?
                <SignUpText onPress={() => nav.navigate(screens.SignUp)}> Sign up</SignUpText>
            </InformationText>
        </SignUpContainer>
    );
};
export default SignUp;
