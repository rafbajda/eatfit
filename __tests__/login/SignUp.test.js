import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import NavigationTestUtils from 'react-navigation/NavigationTestUtils';
import SignUp from '../../app/screens/login/components/SignUp';

describe('SignUp snapshot', () => {
    jest.useFakeTimers();
    beforeEach(() => {
        NavigationTestUtils.resetInternalState();
    });

    it('renders component', async () => {
        const tree = renderer.create(<SignUp />).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
