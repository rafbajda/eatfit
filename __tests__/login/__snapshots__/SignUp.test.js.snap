// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp snapshot renders component 1`] = `
<View
  className="sc-bxivhb sc-gZMcBi fJlXDV"
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
    }
  }
>
  <Text
    className="sc-gqjmRU bPMnPF"
    style={
      Object {
        "color": "#000",
        "fontFamily": "System",
        "fontSize": 16,
      }
    }
    uppercase={false}
  >
    [missing "en.info.dontHaveAccount" translation]
    <Text
      className="sc-VigVT cRRkLs"
      onPress={[Function]}
      style={
        Object {
          "color": "#000",
          "fontFamily": "System",
          "fontSize": 16,
        }
      }
      uppercase={false}
    >
       [missing "en.actions.signUp" translation]
    </Text>
  </Text>
</View>
`;
